events/views.py

```diff
@@ -5,6 +5,7 @@
 from django.shortcuts import redirect
 from django.db.models import Q, Count
 from django.contrib.auth.decorators import login_required
+from django.utils.safestring import mark_safe
```

Comment:
mark_safe could be causing issues. It marks message strings as 'safe to run' in html templates. Could be protection against code injection.